packages/wdio-allure-reporter/src/index.js

```diff
@@ -120,6 +124,20 @@ class AllureReporter extends WDIOReporter {
         currentTest.addLabel('thread', cid)
     }
 
+    getLabels(test){
+        var tags = test.tags
+        var labels = []
+        if (tags != null && tags != undefined){
+            tags.forEach(function (tag) {
+                var label = tag.name.replace(/[@]/, '').split('=')
+                if(label.length == 2){
```

Comment:
It's more solid to use `===` instead of `==`

Reply:
Done